.github/workflows/stale.yml

```diff
@@ -14,22 +14,31 @@ jobs:
       - uses: actions/stale@6f05e4244c9a0b2ed3401882b05d701dd0a7289b # pin@v7
         with:
           # Idle number of days before marking issues stale, set `-1` to disable
-          days-before-issue-stale: 180
+          days-before-issue-stale: 75
 
           # Idle number of days before marking issues stale, set `-1` to disable
-          days-before-pr-stale: -1
+          days-before-pr-stale: 75
 
           # Idle number of days before closing stale issues/PRs
-          days-before-close: 14
+          days-before-close: 21
 
           # Comment on the staled issues
-          stale-issue-message: "This issue has been automatically marked as stale because it has not had recent activity. This is for us to prioritize issues that are still relevant to our community. It will be closed if no further activity occurs within the next 14 days. If this issue is still relevant to you, please leave a comment."
+          stale-issue-message: "This issue has been marked as stale because it requires further input but has not seen activity in the past months. This is for us to prioritize issues that are still relevant and actionable. It will be closed if no further activity occurs within the next 21 days. If this issue is still relevant to you, please help us in gathering the necessary input."
+
+          # Comment on the staled PRs
+          stale-pr-message: "This PR has been marked as stale because it requires further changes but has not seen activity in the past months. This is for us to prioritize PRs that can be reviewed and merged. It will be closed if no further activity occurs within the next 21 days. If you still have interest in this PR, please help us finalizing it."
 
           # Label to apply on staled issues
           stale-issue-label: "type: stale 💤"
 
+          # Label to apply on staled PRs
+          stale-pr-label: "type: stale 💤"
+
           # Reason to use when closing issues
           close-issue-reason: not_planned
 
+          # Labels to check for stale issues/PRs
+          any-of-labels: "needs: changes 🔁,"
```

Comment:
```suggestion
          any-of-pr-labels: "needs: changes 🔁"
          any-of-issue-labels: "needs: discussion 🗣,needs: decision 🗳,needs: information ❓,needs: replication 🔬"
```

Reply:
Adapted some of it, wouldn't include discussion/decision as that's an action we as team need to do, not the user.